src/tests/homework.spec.js

```diff
@@ -0,0 +1,221 @@
+// Homework 1 
+
+import { test, expect } from "@playwright/test";
+
+//return locator objects
+
+async function getNameFieldLocator(page) {
+    return page.locator("#name");
+}
+async function getEmailFieldLocator(page){
+    return page.getByLabel("Email");  
+}
+
+async function getPasswordFieldLocator(page) {
+    return page.getByLabel("Heslo");
+}
+async function getControlPasswordFieldLocator(page) {
+    return page.getByLabel("Kontrola hesla");
+}
+async function getRegistraceButtonLocator (page) {
+    return page.getByRole('button', {name: 'Zaregistrovat'});
+}
+
+async function getCurrentUserLocator(page) 
+{
+    const currentUser = page
+        .locator(".navbar-right")
+        .locator("strong");
+    return currentUser;
+}
+async function getLoginInLocator(page) 
+{
+    const loginInLocator = await page.locator(".navbar-right"); 
+    return loginInLocator;
+}
+
+// return texts of error messages
```

Comment:
U funkce getToastMessageLocator() není nutné použít komentář, je z kódu explicitně jasné co vrací. Také není nutné ukládat do proměnné, klidně to mohlo být jen:
```
async function getToastMessageLocator(page) {
    return await page.locator(".toast-message").textContent();
}
```